ui/plugins/failure_summary/controller.js

```diff
@@ -63,7 +63,21 @@ treeherder.controller('BugsPluginCtrl', [
                             );
                             suggestions.push(suggestion);
                         });
+                        var errors = [];
+                        if (suggestions.length === 0 && artifact_list.length > 1) {
+                            var artifact = _.find(artifact_list, {"name": "text_log_summary"});
```

Comment:
sorry to keep finding things.  ``artifact`` was defined above and is redefined here.  So my linter is complaining.  Would you either use a different var name, or just remove the ``var`` before it?